### PR TITLE
Add speaker bruce lawson

### PR DESCRIPTION
**Pull request by drublic**

### CODE REVIEW

File: lib/views/pages/speakers.html [hunk 1]
```diff
@@ -32,6 +32,21 @@ <h4 class="speaker_title">Amber Wilson</h4>
         </div>
     </figure>
 
+    <figure id="brucelawson" class="speaker speaker--light">
+        <img src="/img/speaker/bruce.jpg" alt="Bruce Lawson, speaker at NightlyBuild Conference 2017" class="speaker_image" />
```
Comment (asciidisco) [inline lib/views/pages/speakers.html]: Beim "alt" text noch MC statt speaker

**Comment by anselmh** [comment]
has conflicts now…

**Review by anselmh — APPROVED** [review]

**Review by asciidisco — APPROVED** [review]

File: lib/views/pages/speakers.html (hunk 1 above)
Comment (drublic) [reply lib/views/pages/speakers.html]: Done.

**Comment by drublic** [comment]
@anselmh fixed.

**Pull request merged**